devel-common/src/tests_common/test_utils/version_compat.py

```diff
@@ -79,4 +91,6 @@ def get_sqlalchemy_version_tuple() -> tuple[int, int, int]:
     "PokeReturnValue",
     "remove_task_decorator",
     "timezone",
+    "AirflowFailException",
+    "AirflowSensorTimeout",
 ]
```

Comment:
Let’s sort this better


```suggestion
__all__ = [
    "AIRFLOW_V_3_0_PLUS",
    "AIRFLOW_V_3_0_1",
    "AIRFLOW_V_3_1_PLUS",
    "AIRFLOW_V_3_2_PLUS",
    "NOTSET",
    "SQLALCHEMY_V_1_4",
    "SQLALCHEMY_V_2_0",
    "XCOM_RETURN_KEY",
    "AirflowFailException",
    "AirflowSensorTimeout",
    "ArgNotSet",
    "PokeReturnValue",
    "remove_task_decorator",
    "timezone",
]
```

Reply:
Let's do this in separate PR to avoid confusion

Reply:
Nope. I will fix this here...

Reply:
Turns out we no longer needs this.